packages/web/src/components/gcds-pagination/gcds-pagination.tsx

```diff
@@ -0,0 +1,336 @@
+      else if (this.currentPage > this.totalPages - 5 && i >= this.totalPages - 6) {
+        if (i == this.totalPages - 1 && (this.currentPage == this.totalPages - 3 || this.currentPage == this.totalPages - 4)) {
+          var breakpoint = "";
+          switch(this.currentPage - i) {
+            case -2:
+              breakpoint = "420"
+              break;
+            case -3:
+              breakpoint = "460"
+              break;
+            case -4:
+              breakpoint = "488"
+              break;
+          }
+          this.listitems.push(<li class={`gcds-pagination-list-mobile-ellipses-${breakpoint}`}><span class="gcds-pagination-list-ellipses">...</span></li>);
+        }
+        this.listitems.push(this.configurePaginationStep(i));
+      }
+      // Previous and next page
+      else if (i >= (this.currentPage-2) && i <= (this.currentPage+2)) {
+        this.listitems.push(this.configurePaginationStep(i));
+      }
+      // Left ellipses
+      else if (!previousEllipses && i < (this.currentPage-2)) {
+        this.listitems.push(<li><span class="gcds-pagination-list-ellipses">...</span></li>);
+        previousEllipses = true;
+      }
+      // Right ellipses
+      else if (!nextEllipses && i > (this.currentPage+1) && i < this.totalPages - 1) {
+        this.listitems.push(<li><span class="gcds-pagination-list-ellipses">...</span></li>);
+        nextEllipses = true;
+      }
+
+      
+    }
+    if (this.currentPage != this.totalPages) {
+      this.listitems.push(this.configurePaginationStep(this.currentPage, "next"));
+      this.mobilePrevNext.push(this.configurePaginationStep(this.currentPage, "next", true));
+    }
+  }
+
+  async componentWillLoad() {
+    // Define lang attribute
+    this.lang = assignLanguage(this.el);
+
+    if (this.display == "list") {
+      this.configureListPagination();
+    }
+  }
+
+  componentDidUpdate() {
+    if (this.display == "list") {
+      this.configureListPagination();
+    }
+  }
+
+  render() {
+    const { display, label, previousHref, previousLabel, nextHref, nextLabel, lang } = this;
+
+    return (
+      <Host
+        role="navigation"
+        aria-label={label}
+      >
+        {display === "list" ?
+          <div>
+            <ul
+              class="gcds-pagination-list"
+            >
+              {this.listitems}
+            </ul>
+            <ul
+              class="gcds-pagination-list-mobile-prevnext"
+            >
+              {this.mobilePrevNext}
+            </ul>
+          </div>
+        :
+          <ul
+            class="gcds-pagination-simple"
+          >
+            <li>
+              <a
+                href={previousHref}
+                aria-label={`${I18N[lang].previousPage}${previousLabel ? `: ${previousLabel}` : ""}`}
+                onClick={(e) => this.onPageChange(e)}
+              >
+                <gcds-icon class="gcds-pagination-arrow-left" name="arrow-left"></gcds-icon>
+                <b>
```

Comment:
Can you switch this to a `<strong>` tag? The `<b>` tag is kinda outdated and shouldn't really be used anymore.